src/backend/libs/efiling-account-client/src/test/java/ca/bc/gov/open/jag/efilingaccountclient/config/AutoConfigurationTest.java

```diff
@@ -20,7 +20,7 @@
     private AutoConfiguration sut;
 
 
-    @Test
+    //@Test
```

Comment:
remove commented line possibly

Reply:
Yeah we had commented a bunch out to make things work :|